### PR TITLE
Install missing lib for sqlite

### PR DESCRIPTION
**Pull request by AaronDewes**

### CODE REVIEW

**Comment by AaronDewes** [comment]
@nolim1t This is quite an important fix, as the containers were broken before

File: 0.21/Dockerfile [hunk 1]
```diff
@@ -205,7 +205,8 @@ RUN apk add --no-cache \
         libevent \
         libsodium \
         libstdc++ \
-        libzmq
+        libzmq \
+        sqlite-libs
```
Comment (lukechilds) [inline 0.21/Dockerfile:209]: Is this definitely the correct package name? You're using `sqlite-libs` here but `sqlite-dev` is used in the build image.

Comment (AaronDewes) [reply 0.21/Dockerfile:209]: Yes, sqlite-dev depends on sqlite-libs, but also contains the C header files, which are only needed at compile-time.

**Comment by nolim1t** [comment]
I've compiled in sqlite on v0.21.0 (now build 19 instead of build 18)

https://hub.docker.com/layers/lncm/bitcoind/v0.21.0-arm64v8-build19/images/sha256-d5e75cf79ab2a9a677279ed97e74ee9c034e3dc2303c5c18a5bf9e1d2007e80b?context=explore

**Comment by nolim1t** [comment]
I'm going to allow this in anyway and do another build (build 20)

**Comment by nolim1t** [comment]
this is being pushed as build 20 as build 19 didn't work

**Pull request merged**